lib/tasks/email_template.rake

```diff
@@ -0,0 +1,19 @@
+# rake db:email_template
+
+# Later, with more templates, email_template can be a namespace,
+# and each template could have separate task
+
+begin
```

Comment:
I don't see any tests for any rake task in LS. Do you want me to make one for this task?